website/blog/2018-12-21-state-of-react-native-community.md

```diff
@@ -0,0 +1,51 @@
+---
+title: The State the React Native Community in 2018
+author: Lorenzo Sciandra
+authorTitle: Core Maintainer & React Native Developer
+authorURL: https://github.com/kelset
+authorImageURL: https://avatars2.githubusercontent.com/u/16104054?s=460&v=4
+authorTwitter: kelset
+category: announcements
+---
+
+This year we made a large number of changes about how we develop and communicate about React Native; we hope that in two years from now we can look back and see that this shift was a turning point for React Native.
+
+A lot of people are excited about the rewrite of React Native's architecture, widely known as [Fabric](https://github.com/react-native-community/discussions-and-proposals/issues/4). Among other things, this will fix fundamental flaws in React Native's architecture and will set up React Native for success in the future together with [JSI and TurboModules](https://github.com/react-native-community/discussions-and-proposals/issues/40).
```

Comment:
```suggestion
A lot of people are excited about the rewrite of React Native's architecture, widely known as [Fabric](https://github.com/react-native-community/discussions-and-proposals/issues/4). Among other things, this will fix fundamental limitations in React Native's architecture and will set up React Native for success in the future together with [JSI and TurboModules](https://github.com/react-native-community/discussions-and-proposals/issues/40).
```

- Switch "flaws" to "limitations"